.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think these files in `.idea` were your own playing around and not meant to be part of this PR - if you meant to include them can you explain what/why?